src/Lykke.AlgoStore.Api/Modules/AlgoApiModule.cs

```diff
@@ -25,6 +26,7 @@ protected override void Load(ContainerBuilder builder)
                 .As<ILog>()
                 .SingleInstance();
 
+            builder.RegisterAlgoInstanceStoppingClient("http://localhost:5000", _log);
```

Comment:
Is this not supposed to be a setting?

Reply:
yes, it is in settings, I missed to remove it from tests

Reply:
This is the setting https://github.com/LykkeCity/Lykke.AlgoStore/pull/181/commits/03d6ef67068877f517fb31c23745964edad774b7#diff-76ad82a2a14ba92714604af32a97e425R82